plugins/org.eclipse.emf.mwe.utils/src/org/eclipse/emf/mwe/utils/StandaloneSetup.java

```diff
@@ -211,11 +213,21 @@ protected File findProjectFileForPossibleClassesFolder(File f) {
 					}
 				}
 			}
+			// gradle(buildship)
+			File grandParentFile = parentFile.getParentFile();
+			if (grandParentFile != null) {
+				if (parentFile.getName().equals("bin")) {
```

Comment:
bin or build?

Reply:
bin. see https://github.com/eclipse/xtext-core/pull/876/files

Reply:
Yes, but other project may use the gradle default. Apart from that, this looks good to me.

Reply:
can we treat this with another issue?